broker/broker-topic-logger/src/main/java/org/codice/ddf/broker/logger/TopicLogger.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.broker.logger;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.builder.RouteBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TopicLogger extends RouteBuilder {
+    private static final Logger LOGGER = LoggerFactory.getLogger(TopicLogger.class);
+
+    private String name = "example.topic";
+
+    private long samplePeriod = 3600L;
+
+    private boolean showHeaders;
+
+    public TopicLogger(CamelContext camelContext) throws Exception {
+        setContext(camelContext);
+    }
+
+    @Override
+    public void configure() throws Exception {
+
+        fromF("sjms:topic:%s", name).routeId(
```

Comment:
Should we make the protocol configurable or can we expect that this will always work?